eyeshade/workers/surveyors.js

```diff
@@ -1,6 +1,7 @@
 const { insertFromVoting, updateBalances } = require('../lib/transaction.js')
 const { mixer } = require('../workers/reports.js')
 
+exports.name = 'surveyors'
```

Comment:
I noticed we have similar lines in many worker files, though I don't know what this does.  Could you explain what these lines are doing?

Reply:
this is a legacy line from the previous implementations that marshal had setup. they can be removed. i was just trying to be consistent until we do so